.travis.yml

```diff
@@ -1,9 +1,13 @@
----
 language: node_js
 node_js:
-- "0.12"
+- '0.12'
+env:
+  global:
+  - CONTENT_STORE_URL: https://deconst.horse:9000/
+    NODE_TLS_REJECT_UNAUTHORIZED: 0
```

Comment:
Unfortunate, but eh, it's deconst.horse. We can :hocho: when we have a real cert.